keyboards/orthodox/keymaps/xyverz/config.h

```diff
@@ -22,7 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifndef CONFIG_USER_H
 #define CONFIG_USER_H
 
-#include "../../config.h"
+#include "../../rev1/config.h"
```

Comment:
QMK automatically includes all the `config.h` files in the build tree, so this line is redundant.

```suggestion
```